src/Take.Elephant.Tests/Sql/DbConnectionDecoratorFacts.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Take.Elephant.Sql;
+using Xunit;
+using NSubstitute;
+using System.Data.Common;
+using System.Threading.Tasks;
+
+namespace Take.Elephant.Tests.Sql
+{
+    public class DbConnectionDecoratorFacts : FactsBase
```

Comment:
This class name should be change since the related class has changed.

Reply:
I really forgot, done